Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "allms"
-version = "0.8.2"
+version = "0.8.3"
```

Comment:
pros: next time they do cargo `upgrade` they'll get this version. 

cons: the new range might be considered an api change, so technically we should bump minor at least?

X stands for a major version. The leftmost number denotes a major version. When you increase the major version number, you increase it by one but you reset both patch version and minor versions to zero. If the current version is 2.6.9 then the next upgrade for a major version will be 3.0.0. Increase the value of X when breaking the existing API.

Y stands for a minor version. It is used for the release of new functionality in the system. When you increase the minor version, you increase it by one but you must reset the patch version to zero. If the current version is 2.6.9 then the next upgrade for a minor version will be 2.7.0. Increase the value of Y when implementing new features in a backward-compatible way.

Z stands for a Patch Versions: Versions for patches are used for bug fixes. There are no functionality changes in the patch version upgrades. If the current version is 2.6.9 then the next version for a patch upgrade will be 2.6.10. There is no limit to these numbers. Increase the value of Z when fixing bugs

Reply:
Theoretically we are adding some new capabilities, but it's all done if a very backwards compatible way so there is absolutely no breaking changes here. People can safely upgrade without any worry.

Also, the new capabilities are not necessarily new features there are more enhancements of the existing feature. Previously you could set the temp directly in constructor, this just offers more control that the temp is in range for different providers.

Given this all, I would probably say it's more of patch of existing capabilities than something that would drive new uses.

Reply:
Alright, hold that thought. Turns out we were currently not setting the temperature for Assistants. I need to add this capability, which will probably mean a new minor. More changes coming to this PR....

Reply:
Ok, I pushed some more changes that add the temperature setting for `Assistants` as well. This probably qualifies as new capabilities so I bumped the minor version to `0.9.0`.